stellarphot/photometry/photometry.py

```diff
@@ -187,8 +196,31 @@ def single_image_photometry(ccd_image, sourcelist, camera, observatory_location,
         raise ValueError(f"input_coordinates ({use_coordinates}) must be either "
                          "'pixel' or 'sky'.")
 
-    # Check CCDData headers before proceeding
+    # Set up logging (retrieve a logger but purge any existing handlers)
+    root_logger = logging.getLogger()
+    for handler in root_logger.root.handlers[:]:
+        root_logger.root.removeHandler(handler)
```

Comment:
I assume this was because of the root handler in `ccdproc`? It seems like removing other handlers might mess with other people's stuff? 

I don't have any better ideas, and am not sure what, if anything, to change here.

Reply:
I got the impression from the documentation that the root handler isn't really supposed to be used if you are working on logging for a specific function/class or package... just by the fact that [the docs promote to automatically set up loggers for specific modules](https://docs.python.org/3/howto/logging.html#advanced-logging-tutorial) saying 

*A good convention to use when naming loggers is to use a module-level logger, in each module which uses logging, named as follows:*
```
logger = logging.getLogger(__name__)
```

That said, I agree, it wasn't necessarily a friendly thing to slam off any root loggers running, since someone might want to import our code into their code... and using the `root` logger is completely acceptable behavior.  But I don't think we can 'disable' the root handler and then re-activate it.

Reply:
Something weird is going on ccdproc uses the thing you suggested when it does logging:

```
logger = logging.getLogger(__name__)
```

Reply:
Maybe one of the packages ccdproc imports, although a brief review of that code doesn't reveal any packages I don't use regularly.  I suspect when you create a logger for a module, since it is a child of the root logger, a root logger must be created... maybe for some reason you are creating the root handler?  I did check and I didn't see any of the mistakes I saw others point out (e.g. using the BasicConfig setup).